docs/docs/methods.md

```diff
@@ -1311,3 +1311,233 @@ Do not add properties which are absent in an object but are present in transform
+  { name: 'Nick Oldman', age: 45 },
+  { name: 'Jack Newton', age: 12 }
+]
+filter(({ age }) => age >= 18, people)  // [{ name: 'Nick Oldman', age: 45 }]
+
+const peopleObj = {
+  'Nick Oldman': 45,
+  'Jack Newton': 12
+}
+filter(age => age >= 18, peopleObj)     // { 'Nick Oldman': 45 }
+```
+
+## `filterT`
+
+Creates a transducer with a filter
+
+```js
+import { filterT } from 'nanoutils'
+
+const pushReducer = (array, value) => {
+  array.push(value)
+  return array
+}
+const isEven = value => value % 2 === 0
+const transducer = filterT(isEven)
+const rootReducer = transducer(pushReducer)
+
+rootReducer([], 1)      // []
+rootReducer([], 2)      // [2]
+rootReducer([2], 3)     // [2]
+rootReducer([2], 4)     // [2, 4]
+rootReducer([2, 4], 5)  // [2, 4]
+```
+
+## `find`
+
+Finds a value in an `array` based on a predicate and returns it (a search is made from a beginning). Otherwise, returns `undefined`
+
+```js
+import { find } from 'nanoutils'
+
+const isDigit = value => value >= 0 && value < 10
+
+find(isDigit, [10, 20, 30])   // undefined
+find(isDigit, [10, 2, 3])     // 2
+```
+
+## `findIndex`
+
+Finds a value in an `array` based on a predicate and returns an index of it (a search is made from a beginning). Otherwise, returns `-1`
+
+```js
+import { findIndex } from 'nanoutils'
+
+const isDigit = value => value >= 0 && value < 10
+
+findIndex(isDigit, [10, 20, 30])    // -1
+findIndex(isDigit, [10, 2, 3])      // 1
+```
+
+## `findLast`
+
+Finds a value in an `array` based on a predicate and returns it (a search is made from an end). Otherwise, returns `undefined`
+
+```js
+import { findLast } from 'nanoutils'
+
+const isDigit = value => value >= 0 && value < 10
+
+findLast(isDigit, [10, 20, 30])   // undefined
+findLast(isDigit, [10, 2, 3])     // 3
+```
+
+## `findLastIndex`
+
+Finds a value in an `array` based on a predicate and returns an index of it (a search is made from an end). Otherwise, returns `-1`
+
+```js
+import { findLastIndex } from 'nanoutils'
+
+const isDigit = value => value >= 0 && value < 10
+
+findLastIndex(isDigit, [10, 20, 30])    // -1
+findLastIndex(isDigit, [10, 2, 3])      // 2
+```
+
+## `flatten`
+
+Returns an `array` with elements of it and its subarrays
+
+```js
+import { flatten } from 'nanoutils'
+
+var array = [1, 2, [3, [4, [5]], 6], [7], 8]
+
+flatten(array)  // [1, 2, 3, 4, 5, 6, 7, 8]
+```
+
+::: tip
+In spite of `flatten`, `unnest` returns an array with unwrapped elements of a depth `1`
+:::
+
+## `flattenObj`
+
+It converts nested object into a flatten one, all keys become paths to repsective values
+
+```js
+import { flattenObj } from 'nanoutils'
+
+const people = {
+  john: {
+    name: 'John Parker',
+    age: 18
+  }
+}
+flattenObj(people)  //  { 'john.name': 'John Parker', 'john.age': 18 }
+```
+
+::: tip
+Remember that `flattenObj` is applied only to `object`s
+
+To apply it to `array`s have a look at [flatten](#flatten)
+:::
+
+## `flip`
+
+Applies a function to a reversed arguments
```

Comment:
I assume it should be told explicit like “...reversed arguments list”, because ”reversed arguments” could be interpreted like “5 -> -5” or another weird way